pkg/balancer-js/src/utils/errors.ts

```diff
@@ -56,6 +56,10 @@ const balancerErrorCodes: Record<string, string> = {
   '328': 'CALLER_IS_NOT_LBP_OWNER',
   '329': 'PRICE_RATE_OVERFLOW',
   '330': 'INVALID_JOIN_EXIT_KIND_WHILE_SWAPS_DISABLED',
+  '331': 'LOWER_GREATER_THAN_UPPER_TARGET',
+  '332': 'UPPER_TARGET_TOO_HIGH',
+  '333': 'UNHANDLED',
```

Comment:
Let's be a bit more descriptive on this one

Reply:
what are you thinking for example? because it is used onJoin, onExit and onInitialized that are not handled

Reply:
renaming to `UNHANDLED_BY_LINEAR_POOL`